src/lib/Components/Inbox/Conversations/Avatar.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react"
+
+import { Text, View } from "react-native"
+
+import styled from "styled-components/native"
+import colors from "../../../../data/colors"
+import fonts from "../../../../data/fonts"
+
+const BackgroundCircle = styled.View`
+  background-color: ${colors["gray-regular"]};
+  height: 30
+  width: 30
+  border-radius: 15
+`
+const Name = styled.Text`
+  color: ${props => (props.user ? "white" : colors["gray-semibold"])};
+  font-family: ${fonts["avant-garde-regular"]};
+  font-size: 10
+  align-self: center
+  margin-top: 8
+`
+
+interface Props {
+  senderName: string
+  isUser: boolean
+}
+
+export default class Avatar extends React.Component<Props, any> {
+  render() {
+    const initials: string = this.props.senderName
+      .split(" ")
+      .map((nameComponent: string) => nameComponent[0])
+      .join("")
+      .substring(0, 3)
+      .toUpperCase()
+    if (this.props.isUser) {
+      return <BackgroundCircle><Name user>{initials}</Name></BackgroundCircle>
```

Comment:
This is the [passing of props with styled components](https://www.styled-components.com/docs/basics#adapting-based-on-props) but it seems to give type-check errors. @sarahscott did you end up using this in one of your other PRs? I remember you, @l2succes, and I talking about it in comments.